frontend/src/alert_configuration/alert_configuration_app.js

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
```

Comment:
Recommendation to split up the components in different files (e.g. Footer.js) - although I think the AppBar can be replaced by the NavBar in the other PR I made - and have one file that puts all the page components together (if that makes sense - let me know if it doesn't!)